projects/storefrontlib/src/cms-components/myaccount/forgot-password/forgot-password.component.ts

```diff
@@ -7,16 +7,16 @@ import { CustomFormValidators } from '../../../shared/utils/validators/custom-fo
   templateUrl: './forgot-password.component.html',
 })
 export class ForgotPasswordComponent implements OnInit {
```

Comment:
Breaking changes to document:
- removed submited

Reply:
Done